packages/client/wallets/aa/src/utils/blockchain.ts

```diff
@@ -75,10 +75,29 @@ export async function verifyMessage({ address, message, signature, chain }: Veri
     });
 }
 
-export function hasEIP1559Support(chain: EVMBlockchainIncludingTestnet) {
-    const chainsNotSupportingEIP1559: EVMBlockchainIncludingTestnet[] = [
+function isPolygonCDK(chain: EVMBlockchainIncludingTestnet) {
+    const polygonCDKchains: EVMBlockchainIncludingTestnet[] = [
         EVMBlockchainIncludingTestnet.ZKYOTO,
+        EVMBlockchainIncludingTestnet.ZKATANA,
         EVMBlockchainIncludingTestnet.ASTAR_ZKEVM,
+        EVMBlockchainIncludingTestnet.HYPERSONIC_TESTNET,
     ];
-    return !chainsNotSupportingEIP1559.includes(chain);
+    return polygonCDKchains.includes(chain);
 }
+
+export function hasEIP1559Support(chain: EVMBlockchainIncludingTestnet) {
```

Comment:
nit: this function is only used once, we can directly use `isPolygonCDK`

Reply:
In the PR I have open that drops `sendTransaction` this'll be deleted so leaving for now!